wiki/Reporting_Bad_Behaviour/en.md

```diff
@@ -1,116 +1,99 @@
----
-needs_cleanup: true
----
-
-<!-- TODO: FAQ-like things and overall it doesn't do a good job of describing the report system first. -->
-
 # Reporting bad behaviour
 
-## What is the report system?
-
-The report system is an important feature to inform the moderators about misbehaving user(s) and for them to take action.
-
-It is not only limited to in-game chat but also in other places such as the forum and wiki itself.
-
-## How to report someone
-
-### Using the !report command in-game
-
-Usage: `!report username [reason]`
-
-Parameters in **[** and **]** are optional.
-
-If the user has a space in their username, it must be replaced with an underscore `_` because the osu! in-game chat is based on [Internet Relay Chat](/wiki/Internet_Relay_Chat).
-
-`!report This User Name is spamming in #osu` turns to `!report This_User_Name is spamming in #osu`
-
-Reports are not case-sensitive so it is okay to use lowercase letters like `this_user_name`.
-
-![The chat box with a !report command](img/report-command.jpg "Example of using the !report command")
-
-### Using the user report button in-game
-
-Reporting the user by this way only needs a type of reason and the custom text, unlike the `!report` command.
-
-![The list of available user options](img/report-user-1.png "Click the Report User button")
-
-![The user report overlay](img/report-user-2.png "Remember to add a reason here!")
+osu! has [a team of moderators](/wiki/People/The_Team/Global_Moderation_Team) who adhere to keeping the game and its ecosystem clean and safe. If you notice rule-breaking behaviour or inappropriate content, report it using one of the methods described below. Alternatively, you may contact any of the moderators online directly if your report wasn't properly handled within a reasonable amount of time — do so only as the last resort.
```

Comment:
one team, group or organisation is still singular

```suggestion
osu! has [a team of moderators](/wiki/People/The_Team/Global_Moderation_Team) that adheres to keeping the game and its ecosystem clean and safe. If you notice rule-breaking behaviour or inappropriate content, report it using one of the methods described below. Alternatively, you may contact any of the moderators online directly if your report wasn't properly handled within a reasonable amount of time — do so only as the last resort.
```

Reply:
hmm, just realised that it's ambiguous whether it's the "team" or the "moderators" doing the adhering, depending on where you put the emphasis while reading, so keep it or change it as you prefer.